app/tag-management/all-tags/all-tags.component.html

```diff
@@ -0,0 +1,14 @@
+<md-card class="default-card">
+  <div class="page-header">
+    <h2>{{ 'all tags' | translate }}</h2>
```

Comment:
In all other components, we have an extra md-card for the title and using `h1` as headline. Please do it like that for your components as well to be consistent:
```
<md-card class="default-card">
   <h1>Title</h1>
</md-card>
```